data/editor.config

```diff
@@ -1051,6 +1056,7 @@
       </type>
       <type id="amenity-biergarten" group="food">
         <include group="poi_internet" />
+        <include field="website_menu" />
```

Comment:
Does it make sense to allow editing cuisine for a biergarten? @organicmaps/translations-de WDYT?

Reply:
According to the wiki, it make sense ^^ 

https://wiki.openstreetmap.org/wiki/Tag:amenity%3Dbiergarten